tools/src/main/java/org/apache/kafka/tools/reassign/Tuple.java

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Let's not include this class in this PR. At this stage it's unclear if it will be needed. If we really need it let's put it in the PR moving the tool.

Reply:
Removed